index.d.ts

```diff
@@ -413,6 +414,7 @@ declare module '@primer/components' {
     block?: boolean
     icon?: ReactComponentLike
     variant?: 'small' | 'large'
+    ref?: HTMLSpanElement
```

Comment:
@smockle does this type definition look right??

Reply:
I think we discussed this as part of https://github.com/github/memex/pull/1171, and I note the latest commit in this PR uses a different type, so I’m going to mark this conversation as “resolved”. If there is more to discuss, though, please feel free to re-open! 😊